arlo_server/audit_boards.py

```diff
@@ -210,7 +211,27 @@ def calculate_risk_measurements(election: Election, round: Round):
         round_contest.is_complete = is_complete
 
 
+def count_audited_votes(election: Election, round: Round):
+    vote_counts = dict(
+        SampledBallotDraw.query.filter_by(round_id=round.id)
+        .outerjoin(SampledBallot)
+        .group_by(SampledBallot.vote)
+        .values(SampledBallot.vote, func.count())
+    )
```

Comment:
is it necessary to do this join, or can you just select from `SampledBallot` ?

Reply:
Also, this query has another flaw: contest choices are not unique across contests, so I need to change this.